Makefile

```diff
@@ -449,7 +449,7 @@ GENERATED_FILES += $(CLI_DIR)/config-generated.go
 $(TARGET_OUTPUT): $(SOURCES) $(GENERATED_FILES) $(MAKEFILE_LIST) | show-summary
 	$(QUIET_BUILD)(cd $(CLI_DIR) && go build $(BUILDFLAGS) -o $@ .)
 
-$(SHIMV2_OUTPUT):
+$(SHIMV2_OUTPUT): $(SOURCES) $(GENERATED_FILES)
```

Comment:
Didn't want `MAKEFILE_LIST` on here as well, as is on other target lines?

Reply:
@grahamwhaley Wasnt sure what the variable was for before, I understand now that it adds a dependency on all make files. I have added that as a dependency.

Reply:
:-) Yeah, I had to go look it up as well :-)